packages/mobile/capacitor/assets/assetgen.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+base="capacitor/assets/$1"
+dest_ios="ios/App/App/Assets.xcassets/AppIcon.appiconset"
+dest_ios_splash="ios/App/App/Assets.xcassets/Splash.imageset"
+dest_android="android/app/src/main/res"
+# http://astroa.physics.metu.edu.tr/MANUALS/ImageMagick-6.2.5/www/command-line-options.html#filter
```

Comment:
If ImageMagick is required for this script, can we add a note in the readme since this will be run with `yarn build`?

Reply:
yes will do, will also add some more instructions for the new scripts on `package.json`

Reply:
This can be a separate PR